Fix help for utf8-strict-mode (#8572)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# (experimental) Enable UTF-8 strict mode. Allows UTF-8 characters in the
# matchers for routes and inhibition rules, in silences, and in the labels for
# alerts. It is recommended that all tenants run the `migrate-utf8` command in

> # mimirtool before enabling this mode as otherwise some tenant configurations
> # might fail to load. Tenants with incompatible configurations can be identified
> # in Mimir server logs searching for lines containing `Alertmanager is moving to
> # a new parser for labels and matchers, and this input is incompatible.`. Tenant

Reviewer: ```suggestion
# mimirtool before enabling this mode. Otherwise, some tenant configurations
# might fail to load. To identify tenants with incompatible configurations, you can search
# Mimir server logs for lines containing `Alertmanager is moving to
# a new parser for labels and matchers, and this input is incompatible.`.
```

> # configurations that are valid but contain ambiguous matchers can be found
> # searching for log lines containing `Matchers input has disagreement`. Each log

Reviewer: ```suggestion
# To find tenant configurations that are valid but contain ambiguous matchers, 
# search for log lines containing `Matchers input has disagreement`. Each log
```

# line includes the invalid input, a suggestion on how to fix the input, and the

> # ID of the affected tenant. Mimir must be run with debug level logging enabled.
> # More information about this can be found at

Reviewer: ```suggestion
# ID of the affected tenant. To use this mode, you must run Mimir with debug-level logging enabled.
# For more information, refer to
```

Author: Nit: Debug level logging must be enabled to find affected tenants, not use the mode. Does that change the phrasing at all?

Reviewer: Yes, thank you for commenting. Updated my suggestion :)

> # https://prometheus.io/docs/alerting/latest/configuration/#label-matchers.

Reviewer 2: Minor drive-by comment: The link doesn't work because of the fullstop, FYI
```suggestion
# https://prometheus.io/docs/alerting/latest/configuration/#label-matchers
```

Author: Hmm, it works for me?

Author: Oh I see, the jump to `#label-matchers` doesn't work. Will fix! Thanks @tomratcliffe!

Author: Linter might complain though, let's see...

Author: I think this is fine, the other links have a full stop at the end?

# (experimental) Enable UTF-8 strict mode. Allows UTF-8 characters in the
# matchers for routes and inhibition rules, in silences, and in the labels for
# alerts. It is recommended that all tenants run the `migrate-utf8` command in
# mimirtool before enabling this mode. Otherwise, some tenant configurations
# might fail to load. To identify tenants with incompatible configurations,
# search Mimir server logs for lines containing `Alertmanager is moving to a new
# parser for labels and matchers, and this input is incompatible.`. To find
# tenant configurations that are valid but contain ambiguous matchers search for
# log lines containing `Matchers input has disagreement`. Each log line includes
# the invalid input, a suggestion on how to fix the input (excluding ambiguous

> # matchers as these require manual correction), and the ID of the affected

Reviewer: ```suggestion
# matchers, as these require manual correction), and the ID of the affected
```

> # tenant. Mimir must be run with debug level logging enabled otherwise these
> # lines will not be logged. For more information, refer to

Reviewer: ```suggestion
# tenant. You must run Mimir with debug-level logging enabled. Otherwise, these
# lines aren't logged. For more information, refer to
```
